### ISSUE TITLE
Every track's album name starts with "from "

### ISSUE THREAD

**Issue by JanneHarju** [masterT/bandcamp-scraper#23]
If returned type is track then album property starts with "from ". It is coming from this kind of text: "from Pikaboo Playlist, Vol. 1 by OVERWERK"
Example picture here:
![image](https://user-images.githubusercontent.com/17338490/34122739-1c99b64e-e436-11e7-89ff-268f6c84e1d9.png)
Example search here: https://bandcamp.com/search?q=overwerk

**Comment by masterT** [comment]
Damn, I'll take a look!

**Comment by masterT** [comment]
Yep, search for `{ query: 'overwerk' }`, got the results:

```
[ { type: 'artist',
    name: 'OVERWERK',
    url: 'https://overwerk.bandcamp.com',
    imageUrl: 'https://f4.bcbits.com/img/0008153676_23.jpg',
    tags: [ 'electronica', 'dance', 'Electronic', 'electrohouse', 'pop' ],
    genre: ' Electronic',
    location: 'London, Ontario' },
  { type: 'album',
    name: 'Conquer',
    url: 'https://overwerk.bandcamp.com/album/conquer',
    imageUrl: 'https://f4.bcbits.com/img/a0141415591_7.jpg',
    tags:
     [ 'electrohouse',
       'dance',
       'London',
       'Ontario',
       'electronica',
       'New',
       'OVERWERK',
       'pop',
       'ConquerEP',
       'Album',
       'Canada',
       'Fabrik',
       '2013',
       'Electronic' ],
    releaseDate: '27 November 2013',
    numTracks: 4,
    numMinutes: 22,
    artist: 'OVERWERK' },
  { type: 'track',
    name: 'House (feat. Nick Nikon)',
    url: 'https://edmdistrict.bandcamp.com/track/house-feat-nick-nikon',
    imageUrl: 'https://f4.bcbits.com/img/a0303461103_7.jpg',
    tags: [],
    releaseDate: '20 June 2014',
    album: 'Pikaboo Playlist, Vol. 1',
    artist: 'OVERWERK' },
  { type: 'track',
    name: 'Harder, Over, Faster, 12:30',
    url: 'https://soundtrakno.bandcamp.com/track/harder-over-faster-12-30',
    imageUrl: 'https://f4.bcbits.com/img/a2294734149_7.jpg',
    tags: [],
    releaseDate: '15 December 2013',
    album: 'Over',
    artist: 'Daft Punk vs Overwerk' },
  { type: 'fan',
    name: 'rwerkhoven',
    url: 'https://bandcamp.com/rwerkhoven',
    imageUrl: 'https://f4.bcbits.com/img/0010745978_23.jpg',
    tags: [],
    genre: ' Metal' },
  { type: 'track',
    name: 'Kiovskerwerk',
    url: 'https://womba.bandcamp.com/track/kiovskerwerk',
    imageUrl: 'https://f4.bcbits.com/img/a2558835520_7.jpg',
    tags: [],
    releaseDate: '08 July 2006',
    album: 'Evidence of Letta',
    artist: 'WOMBA' },
  { type: 'track',
    name: 'Werkin Overtime',
    url: 'https://goodintent.bandcamp.com/track/werkin-overtime',
    imageUrl: 'https://f4.bcbits.com/img/a1516876153_7.jpg',
    tags: [],
    releaseDate: '31 August 2014',
    album: 'Good Intent Archive I',
    artist: 'Good Intent' },
  { type: 'track',
    name: 'Sterken Overwinnen',
    url: 'https://giorockmore.bandcamp.com/track/sterken-overwinnen',
    imageUrl: 'https://f4.bcbits.com/img/a3379418722_7.jpg',
    tags: [],
    releaseDate: '15 May 2015',
    album: 'Underworld',
    artist: 'Gio Rockmore' },
  { type: 'track',
    name: 'Sterken Overwinnen',
    url: 'https://giorockmore1.bandcamp.com/track/sterken-overwinnen',
    imageUrl: 'https://f4.bcbits.com/img/a1522904786_7.jpg',
    tags: [],
    releaseDate: '15 May 2015',
    album: 'Underworld',
    artist: 'Gio Rockmore' },
  { type: 'track',
    name: 'Onverwerkt Verleden',
    url: 'https://seinsvergessenheit.bandcamp.com/track/onverwerkt-verleden',
    imageUrl: 'https://f4.bcbits.com/img/a0234920556_7.jpg',
    tags: [],
    releaseDate: '01 December 2009',
    album: 'Verzet & Vergelding',
    artist: 'SeinsvergessenheiT' },
  { type: 'track',
    name: 'Twerking Takes Over',
    url: 'https://dubau5.bandcamp.com/track/twerking-takes-over',
    imageUrl: 'https://f4.bcbits.com/img/a1323914208_7.jpg',
    tags: [],
    releaseDate: '15 November 2015',
    album: 'Apocalyptic Apathy',
    artist: 'dubau5' } ]
```

**Comment by masterT** [comment]
🔧 I fixed it, now it returns:

```
[ { type: 'artist',
    name: 'OVERWERK',
    url: 'https://overwerk.bandcamp.com',
    imageUrl: 'https://f4.bcbits.com/img/0008153676_23.jpg',
    tags: [ 'electronica', 'dance', 'Electronic', 'electrohouse', 'pop' ],
    genre: ' Electronic',
    location: 'London, Ontario' },
  { type: 'album',
    name: 'Conquer',
    url: 'https://overwerk.bandcamp.com/album/conquer',
    imageUrl: 'https://f4.bcbits.com/img/a0141415591_7.jpg',
    tags:
     [ 'electrohouse',
       'dance',
       'London',
       'Ontario',
       'electronica',
       'New',
       'OVERWERK',
       'pop',
       'ConquerEP',
       'Album',
       'Canada',
       'Fabrik',
       '2013',
       'Electronic' ],
    releaseDate: '27 November 2013',
    numTracks: 4,
    numMinutes: 22,
    artist: 'OVERWERK' },
  { type: 'track',
    name: 'House (feat. Nick Nikon)',
    url: 'https://edmdistrict.bandcamp.com/track/house-feat-nick-nikon',
    imageUrl: 'https://f4.bcbits.com/img/a0303461103_7.jpg',
    tags: [],
    releaseDate: '20 June 2014',
    album: 'Pikaboo Playlist, Vol. 1',
    artist: 'OVERWERK' },
  { type: 'track',
    name: 'Harder, Over, Faster, 12:30',
    url: 'https://soundtrakno.bandcamp.com/track/harder-over-faster-12-30',
    imageUrl: 'https://f4.bcbits.com/img/a2294734149_7.jpg',
    tags: [],
    releaseDate: '15 December 2013',
    album: 'Over',
    artist: 'Daft Punk vs Overwerk' },
  { type: 'fan',
    name: 'rwerkhoven',
    url: 'https://bandcamp.com/rwerkhoven',
    imageUrl: 'https://f4.bcbits.com/img/0010745978_23.jpg',
    tags: [],
    genre: ' Metal' },
  { type: 'track',
    name: 'Kiovskerwerk',
    url: 'https://womba.bandcamp.com/track/kiovskerwerk',
    imageUrl: 'https://f4.bcbits.com/img/a2558835520_7.jpg',
    tags: [],
    releaseDate: '08 July 2006',
    album: 'Evidence of Letta',
    artist: 'WOMBA' },
  { type: 'track',
    name: 'Werkin Overtime',
    url: 'https://goodintent.bandcamp.com/track/werkin-overtime',
    imageUrl: 'https://f4.bcbits.com/img/a1516876153_7.jpg',
    tags: [],
    releaseDate: '31 August 2014',
    album: 'Good Intent Archive I',
    artist: 'Good Intent' },
  { type: 'track',
    name: 'Sterken Overwinnen',
    url: 'https://giorockmore.bandcamp.com/track/sterken-overwinnen',
    imageUrl: 'https://f4.bcbits.com/img/a3379418722_7.jpg',
    tags: [],
    releaseDate: '15 May 2015',
    album: 'Underworld',
    artist: 'Gio Rockmore' },
  { type: 'track',
    name: 'Sterken Overwinnen',
    url: 'https://giorockmore1.bandcamp.com/track/sterken-overwinnen',
    imageUrl: 'https://f4.bcbits.com/img/a1522904786_7.jpg',
    tags: [],
    releaseDate: '15 May 2015',
    album: 'Underworld',
    artist: 'Gio Rockmore' },
  { type: 'track',
    name: 'Onverwerkt Verleden',
    url: 'https://seinsvergessenheit.bandcamp.com/track/onverwerkt-verleden',
    imageUrl: 'https://f4.bcbits.com/img/a0234920556_7.jpg',
    tags: [],
    releaseDate: '01 December 2009',
    album: 'Verzet & Vergelding',
    artist: 'SeinsvergessenheiT' },
  { type: 'track',
    name: 'Twerking Takes Over',
    url: 'https://dubau5.bandcamp.com/track/twerking-takes-over',
    imageUrl: 'https://f4.bcbits.com/img/a1323914208_7.jpg',
    tags: [],
    releaseDate: '15 November 2015',
    album: 'Apocalyptic Apathy',
    artist: 'dubau5' } ]
```

**Comment by masterT** [comment]
Fixed in version bandcamp-scraper@1.2.2 🎉 !
